extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibConfig.java

```diff
@@ -44,5 +45,17 @@
      * Custom labels to add to the generated image
      */
     @ConfigItem
-    Map<String, String> labels;
+    public Map<String, String> labels;
+
+    /**
+     * The username to use to authenticate with the registry used to pull the base JVM image
+     */
+    @ConfigItem
+    public Optional<String> baseRegistryUsername;
+
+    /**
+     * The password to use to authenticate with the registry used to pull the base JVM image
+     */
+    @ConfigItem
+    public Optional<String> baseRegistryPassword;
```

Comment:
Why the `base` prefix here? Just curious

Reply:
It's not great for sure, but I added it to keep consistency with the `baseJvmImage` and `baseNativeImage` properties.
Does that sound good?

Reply:
I'd just go with `registryUsername` and `registryPassword` then. The `base` in this case doesn't make much sense, I think

Reply:
Nevermind my previous comment, I see there is an username and password in the container-image config

Reply:
Exactly :)